test/index.spec.ts

```diff
@@ -43,7 +44,7 @@ describe('Index', () => {
     const wrappedInbound = pbStream(inbound.conn)
     const wrappedOutbound = pbStream(outbound.conn)
 
-    wrappedOutbound.writeLP(Buffer.from('test'))
+    wrappedOutbound.writeLP(uint8ArrayFromString('test'))
```

Comment:
This was necessary for the browser tests to pass for me as `Buffer` is not in the browser global scope.